lib/json-schema/util/uri.rb

```diff
@@ -54,6 +55,11 @@ def self.file_uri(uri)
         Addressable::URI.convert_path(parsed_uri.path)
       end
 
+      def self.fake_uri(string)
```

Comment:
I agree that it should rather have hash in the name than fake. But actually I would not call it `hash_uri` because the method itself does not hash URIs but is used to hash a schema. So maybe `hash_schema` would be a better name. But now it actually does not care about whether the input is a schema or not, it takes a string and returns a URI created from the hash of that string. So maybe `uri_from_sha1` oder `uri_from_sha1_hash` or something like that would be more precise. But also more verbose. I'm personally getting more and more a fan of being as precise as possible with naming methods as it eases reasoning about code (we likely have some worse candidates here). What do you think?